cmd/caddy/flag.go

```diff
@@ -4,33 +4,19 @@ import (
 	"flag"
 
 	"github.com/caddyserver/ingress/internal/caddy"
-	"github.com/sirupsen/logrus"
 )
 
 func parseFlags() caddy.ControllerConfig {
-	var email string
-	flag.StringVar(&email, "email", "", "the email address to use for requesting tls certificates if automatic https is enabled.")
-
 	var namespace string
 	flag.StringVar(&namespace, "namespace", "", "the namespace that you would like to observe kubernetes ingress resources in.")
 
-	var enableAutomaticTLS bool
-	flag.BoolVar(&enableAutomaticTLS, "tls", false, "defines if automatic tls should be enabled for hostnames defined in ingress resources.")
-
-	var tlsUseStaging bool
-	flag.BoolVar(&tlsUseStaging, "tls-use-staging", false, "defines if the lets-encrypt staging server should be used for testing the provisioning of tls certificates.")
+	var configMapName string
+	flag.StringVar(&configMapName, "config-map", "", "defines the config map name from where to load global options")
```

Comment:
Should we define also the configmap namespace or we suppose that is in the same namespace that we use for the caddy ingress controller?

Reply:
I think the configmap should always be in the controller namespace. The controller will only listen for it in its own namespace anyway 🙂